script.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function(){
+    var postTemplate = $('#post').html();
+    var output = $('.container');
+    
+    $.ajax({
+        url: 'https://jsonplaceholder.typicode.com/posts',
+        dataType:'json',
+        type: 'get',
+        success: function(data) {
+            $(data).each(function (index, value) {
+                $.get('/templates/postTemplate.mst', function(template){
```

Comment:
We don't want to http get template in the loop. Please check network tab in dev tools in the web browser